### PR TITLE
Switch to 100col to make starfish follow slicedimage. 

### PR DESCRIPTION
**Pull request by ambrosejcarr**
We never actually finished doing this. Now we have. :)

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
Closes #291

**Comment by ttung** [comment]
Lol who's following whom? They both claim to follow the other.

**Comment by ambrosejcarr** [comment]
Who knows! 🙃

File: examples/build_sample_experiment.py [hunk 1]
```diff
@@ -45,9 +46,11 @@ def __call__(self, spec_json):
...
             "--{}-dimensions".format(aux_image_name),
             type=StarfishIndex(),
-            help="Dimensions for the {} images.  Should be a json dict, with {}, {}, and {} as the possible keys.  The "
-                 "value should be the shape along that dimension.  If a key is not present, the value is assumed to be "
-                 "0.".format(aux_image_name, Indices.ROUND, Indices.CH, Indices.Z))
+            help="Dimensions for the {} images.  Should be a json dict, with {}, {}, and {} as the "
+                 "possible keys.  The value should be the shape along that dimension.  If a key is "
+                 "not present, the value is assumed to be 0.".format(
+                aux_image_name, Indices.ROUND, Indices.CH, Indices.Z))
+        # TODO ttung: parser.add_argument doesn't return anything, so this seems odd
```
Comment (ttung) [inline examples/build_sample_experiment.py]: It actually does (the action object).  I guess one could say it's not documented...

Comment (ambrosejcarr) [reply examples/build_sample_experiment.py]: Got it. I will remove the #TODO. Thanks.

**Pull request merged**